behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -0,0 +1,41 @@
+from helper.constants import CLA_FALA_URL
+
+from behave import step
+
+
+@step("I am on the Find a legal aid adviser homepage")
+def step_impl_homepage(context):
+    homepage_url = f"{CLA_FALA_URL}"
+    context.helperfunc.open(homepage_url)
+    title_xpath = context.helperfunc.find_by_xpath("//html/body/div/main/div/div/h1")
+    assert title_xpath is not None
+    strip_text = title_xpath.text.replace("\n", " ")
+    assert strip_text, "Find a legal aid adviser or family mediator"
```

Comment:
Can you add this string `Find a legal aid adviser or family mediator` to constants.py file as you can reuse the string value check in your search results page, when you check the header again.